Makefile

```diff
@@ -52,13 +100,50 @@ docker:
 	docker build -t spacetx/starfish .
 	docker run -ti --rm spacetx/starfish build --fov-count 1 --hybridization-dimensions '{"z": 1}' /tmp/
 
+help-integration:
+	$(call print_help, slow, alias for 'fast run_notebooks docker')
+	$(call print_help, run_notebooks, run all files matching 'notebooks/py/*.py')
+	$(call print_help, docker, build docker and run a simple container)
+
+.PHONY: slow docker
+#
+##############################################################
+
+### INSTALL ##################################################
+#
 install-src:
 	pip install --force-reinstall --upgrade -r REQUIREMENTS-DEV.txt -r REQUIREMENTS-NOTEBOOK.txt
 	pip install -e .
 	pip freeze
 
 install-pypi:
-	pip install -r REQUIREMENTS-NOTEBOOK.txt
-	pip install starfish
-
-.PHONY: all fast lint lint-non-init lint-init test mypy refresh_all_requirements slow docker install-dev install-pypi
+	pip install -r REQUIREMENTS-NOTEBOOK.txt starfish
+
+install-travis: install-travis-$(TRAVIS_EVENT_TYPE)
+
+install-travis-cron: install-pypi
+install-travis-: install-src
```

Comment:
What is this for?

Reply:
For when `TRAVIS_EVENT_TYPE` is not defined.